genienlp/paraphrase/run_lm_finetuning.py

```diff
@@ -435,7 +473,7 @@ def parse_argv(parser):
                         help="Epsilon for Adam optimizer.")
     parser.add_argument("--max_grad_norm", default=1.0, type=float,
                         help="Max gradient norm.")
-    parser.add_argument("--num_train_epochs", default=1.0, type=float,
+    parser.add_argument("--num_train_epochs", default=1, type=int,
```

Comment:
This eliminates the option to train for e.g. 1.5 epochs (I'm not sure if it worked properly before or not), but it is fine since we never used it.